contract/gql_mutations/update_contract.py

```diff
@@ -0,0 +1,42 @@
+from contract.gql_mutations import ContractInputType, ContractNotExistException
+from contract.gql_mutations.utils import update_or_create_contract
+from contract.models import Contract
+from core.schema import OpenIMISMutation
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import ValidationError, PermissionDenied
+
+
+class UpdateContractMutation(OpenIMISMutation):
+    _mutation_class = "UpdateContractMutation"
+
+    class Input(ContractInputType):
+        pass
+
+    @classmethod
+    def _validate_mutation(cls, user, **data):
+        if type(user) is AnonymousUser or not user.id:
+            raise ValidationError("mutation.authentication_required")
+
+        contract_uuid = data['uuid']
```

Comment:
You suppose to have uuid in the data.

Reply:
mutation data has all the data sent by fe if I am not mistaken